client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -97,49 +93,35 @@ class PurchasesListing extends Component {
 
 	getProductPurchases() {
 		return (
-			filter(
-				this.props.purchases,
-				( purchase ) =>
-					purchase.active && ( isJetpackProduct( purchase ) || isJetpackSearch( purchase ) )
-			) ?? null
+			this.props.purchases?.filter(
+				( purchase ) => purchase.active && isJetpackProduct( purchase )
+			) ?? []
 		);
 	}
 
 	getTitle( purchase ) {
-		const { currentPlanSlug } = this.props;
-
-		if ( isJetpackSearch( purchase.productSlug ) ) {
-			return getJetpackProductDisplayName( purchase );
-		}
+		const { currentPlan } = this.props;
 
 		if ( isJetpackProduct( purchase ) ) {
 			return getJetpackProductDisplayName( purchase );
 		}
 
-		if ( currentPlanSlug ) {
-			const planObject = getPlan( currentPlanSlug );
+		if ( currentPlan ) {
+			const planObject = getPlan( currentPlan.productSlug );
 			return planObject.getTitle();
 		}
 
 		return null;
 	}
 
-	getTagline( purchase ) {
-		const { currentPlanSlug, translate } = this.props;
-
-		if ( isJetpackProduct( purchase ) ) {
-			return getJetpackProductTagline( purchase );
-		}
-
-		if ( isJetpackSearch( purchase ) ) {
-			return getJetpackProductTagline( 'search' );
-		}
```

Comment:
These checks are unnecessary since the list of products was already filtered by `isJetpackProduct`. Also, `isJetpackProduct` returns true for Jetpack Search, so we never enter he `isJetpackSearch` if statement.